tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -993,7 +946,7 @@ public function jsonRespondedShouldContain(
 					[
 						"code" => "%user_id%",
 						"function" =>
-						[$this, "getUserIdByUserName"],
+						[$this->graphContext, "getUserIdByUserName"],
```

Comment:
I get error here: `Fatal error: Typed property GraphContext::$featureContext must not be accessed before initialization (Behat\Testwork\Call\Exception\FatalThrowableError)`
I don't understand what I need do. do you have ideas?

Reply:
I think the issue is that in `getUserIdByUserName` you are trying to access `featureContext`, but that is only initiated in the `before()` function of `graphContext`, that might not have happened at that point